packages/lib-sourcify/test/verification.spec.ts

```diff
@@ -1,1318 +0,0 @@
-      );
-      const { match, contractAddress } = await deployCheckAndVerify(
-        contractFolderPath,
-        sourcifyChainHardhat,
-        signer,
-      );
-      expectMatch(match, 'perfect', contractAddress);
-    });
-
-    /* it('should verify a contract created by a factory contract and has immutables', async () => {
-      const deployValue = 12345;
-      const childFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutable',
-        'Child'
-      );
-      const factoryFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutable',
-        'Factory'
-      );
-      const [factoryAddress] = await deployFromAbiAndBytecode(
-        localProvider,
-        factoryFolderPath,
-        signer,
-        [deployValue]
-      );
-
-      // Deploy the child by calling the factory
-      const txReceipt = await callContractMethodWithTx(
-        localProvider,
-        factoryFolderPath,
-        factoryAddress,
-        'deploy',
-        signer,
-        [deployValue]
-      );
-      const childAddress = txReceipt.events.Deployment.returnValues[0];
-      const abiEncoded = localProvider.eth.abi.encodeParameter(
-        'uint',
-        deployValue
-      );
-      const match = await checkAndVerifyDeployed(
-        childFolderPath,
-        sourcifyChainGanache,
-        childAddress,
-        {
-          abiEncodedConstructorArguments: abiEncoded,
-        }
-      );
-
-      expectMatch(match, 'perfect', childAddress);
-    }); */
-
-    /* it('should verify a contract created by a factory contract and has immutables without constructor arguments but with msg.sender assigned immutable', async () => {
-      const childFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutableWithoutConstrArg',
-        'Child'
-      );
-      const factoryFolderPath = path.join(
-        __dirname,
-        'sources',
-        'FactoryImmutableWithoutConstrArg',
-        'Factory'
-      );
-      const [factoryAddress] = await deployFromAbiAndBytecode(
-        localProvider,
-        factoryFolderPath,
-        signer,
-        []
-      );
-
-      // Deploy the child by calling the factory
-      const txReceipt = await callContractMethodWithTx(
-        localProvider,
-        factoryFolderPath,
-        factoryAddress,
-        'createChild',
-        signer,
-        []
-      );
-      const childAddress = txReceipt.events.ChildCreated.returnValues[0];
-      const match = await checkAndVerifyDeployed(
-        childFolderPath,
-        sourcifyChainGanache,
-        childAddress,
-        {
-          msgSender: factoryAddress,
-        }
-      );
-
-      expectMatch(match, 'perfect', childAddress);
-    });
-    */
-    it('should fully verify a contract which is originally compiled and deployed with Unix style End Of Line (EOL) source code, but being verified with Windows style (CRLF) EOL source code', async () => {
```

Comment:
Is there a reason why we don't have this test anymore?

Reply:
We replaced it with "should generate variations with different line endings", but I see now the test is not complete

Reply:
We decided to not implement it